### ISSUE TITLE
Update reference

### ISSUE THREAD

**Issue by WenceyWang** [bcuff/AutoLazy#10]
1>MSBUILD : error : Fody: The weaver assembly 'AutoLazy.Fody, Version=2.2.1.0, Culture=neutral, PublicKeyToken=null' references an out of date version of Mono.Cecil.dll. Expected strong name token of '1C-A0-91-87-7D-12-CA-03' but got '50-CE-BF-1C-CE-B9-D0-5E'. The weaver needs to update to at least version 3.0 of FodyHelpers.

**Comment by tom-englert** [comment]
Yeah, this must be updated to support Fody 3.*!

**Comment by bcuff** [comment]
Yes. There are quite a few changes in the way that fody packages work now so it was more work expected. I do plan to update though.

**Comment by tom-englert** [comment]
@bcuff any progress on this?
